src/Client/index.ts

```diff
@@ -171,9 +174,10 @@ export class Client {
   ) {
     this.#throwOnAppendFailure = throwOnAppendFailure;
     this.#connectionSettings = connectionSettings;
+    this.#insecure = !!channelCredentials.insecure;
```

Comment:
isn't that line a double negative?

Reply:
I think that's to make sure that `undefined` and `null` will be properly changed into boolean value.